python/ray/_private/serialization.py

```diff
@@ -231,6 +233,36 @@ def _deserialize_actor_died_error(self, data, metadata_fields):
                 cause=ray_error_info.actor_died_error.actor_died_error_context
             )
 
+    def _deserialize_oom_error(self, data, metadata_fields):
```

Comment:
By the way, are these methods used? Also wasn't sure why they return `RayActorErrors`?